controllers/tuf/tuf_controller.go

```
@@ -121,22 +121,20 @@ func (r *TufReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.R
 
 		if a.CanHandle(target) {
 			newTarget, err := a.Handle(ctx, target)
-			if err != nil {
-				if newTarget != nil {
-					_ = r.Status().Update(ctx, newTarget)
+			if newTarget != nil {
+				if equality.Semantic.DeepDerivative(target.Spec, newTarget.Spec) {
+					_ = r.Update(ctx, target.DeepCopy())
```

we should not ignore errors, minimaly we should return reconcilation results as failure